spec/annotator-content-analysis_spec.js

```diff
@@ -1,6 +1,33 @@
+const options = {
+  dataUrl: '../demo/tua.json',
+  templates: {
+    form: '/templates/form.handlebars',
+    topic: '/templates/topic.handlebars',
+    question: '/templates/question.handlebars',
+    answer: '/templates/answer.handlebars'
+  }
+};
+
+
+
 describe('Annotator Custom Editor ', function() {
-  return describe('On loading the module', function() {
-    return it('should set the data', function() {
+  const analyzer = annotatorContentAnalysis(options);
+  analyzer.start();
+
+  describe('On loading the module', () => {
+    it('exists', () => expect(analyzer).to.exist);
+  });
+
+  describe('has the methods we want', () => {
+
+    it('should set the data', () => {
+
+      // do stuff with analyzer?
+      console.log(analyzer);
+      expect(analyzer).to.equal('')
```

Comment:
let's remove this line since it's causing the tests to fail and just leave it as such for now.

going forward, we can create an annotation fixture and pass it into `analyzer.beforeAnnotationCreated(xxx)`. doing so now causes the module to log the survey to the page which is the first affirmation that we need to decouple the data presentation from the data generation. thank you tests!